tests/components/tibber/test_config_flow.py

```diff
@@ -0,0 +1,69 @@
+"""Tests for Tibber config flow."""
+import pytest
+
+from homeassistant.components.tibber.const import DOMAIN
+from homeassistant.const import CONF_ACCESS_TOKEN
+
+from tests.async_mock import AsyncMock, MagicMock, PropertyMock, patch
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture(name="tibber_setup", autouse=True)
+def tibber_setup_fixture():
+    """Patch tibber setup entry."""
+    with patch("homeassistant.components.tibber.async_setup_entry", return_value=True):
+        yield
+
+
+async def test_show_config_form(hass):
+    """Test show configuration form."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": "user"}
+    )
+
+    assert result["type"] == "form"
+    assert result["step_id"] == "user"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5460460)